core/src/main/java/hudson/model/TimeZoneProperty.java

```diff
@@ -39,11 +39,6 @@ private TimeZoneProperty() {
         this(null);
     }
 
-    @Override
-    public void save() throws IOException {
-        user.save();
```

Comment:
This caused null pointers in jcasc as there's a transient user reference set on the property that can't be set through regular databinding,

I haven't seen this save pattern on any other user property and it works fine without it

I've tested from the UI and restarted jenkins and it is all fine, (and through jcasc)

note that this class is restricted no external use so evolving it is fine
class references:
https://github.com/search?q=org%3Ajenkinsci+TimeZoneProperty&type=Code